modules/representation_management/spec/requests/representation_management/v0/pdf_generator_2122_swagger_spec.rb

```diff
@@ -8,6 +8,14 @@
                                       type: :request do
   path '/representation_management/v0/pdf_generator2122' do
     post('Generate a PDF for form 21-22') do
+      before do
+        # Mock the AccreditedOrganization lookup
+        allow(AccreditedOrganization)
```

Comment:
This mock can be removed by doing something like: `create(:accredited_organization, id: '6f76b9c2-2a37-4cd7-8a6c-93a0b3a73943', name: 'Veterans Organization')` and setting the support file's organization id to the same uuid. `SecureRandom.uuid` will get you a uuid if you want a different one than what I used.